src/viam/sdk/registry/registry.hpp

```diff
@@ -95,6 +95,8 @@ class ModelRegistration {
     std::function<std::shared_ptr<Resource>(Dependencies, ResourceConfig)> construct_resource;
 
     /// @brief Validates a resource config.
+    /// @return a list of the resource's implicit dependencies.
```

Comment:
@bhaney I opted for only updating the documentation here, since it's both in code and our cpp.viam.dev documentation, and I didn't want to have duplicative comments in two locations within the same file. Let me know if you'd like it above as well!